lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -973,12 +973,6 @@ ParseResult FIRParser::parseType(FIRRTLType &result, const Twine &message) {
           if (parseType(type, "expected bundle field type"))
             return failure();
 
-          // We require that elements of aggregates themselves
```

Comment:
This makes `loc` above unused, generating warning, please remove while touching this :+1: .

Reply:
I looked when I did that to see how painful it would be to get loc into the op instead of dropping it.